woocommerce/woocommerce-gutenberg-products-block
# Next Previous Button block - Add layout support

## What

This PR enables the layout property. In this way, it is possibile change the vertical alignment of the next and previous buttons. 
## Why

In this way, block themes can change the button's position via `theme.json`.

The feature is full working, we should spend some time on the following:

- Understand why all the controls are visible even if the they should not be visible according with the block.json configuration.
- ~~Ensure that it works with WordPress 6.2.~~ https://github.com/woocommerce/woocommerce-blocks/pull/10668#issuecomment-1695576831
- Add E2E tests.




## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Open the Site Editor and edit the Single Product Template.
2. Add the Product Gallery.
3. Focus on the Next And Previous Button block.
4. Change the vertical alignment.
5. Save.
6. Visit a product page.
7. Ensure that the vertical alignment matches.
8. Install WordPress 6.2.
9. Open the Site Editor and edit the Single Product Template.
10. Focus on the Next And Previous Button block.
11. Ensure that the Vertical Alignment settings aren't visible.


## Screenshots or screencast




https://github.com/woocommerce/woocommerce-blocks/assets/4463174/8b1bd4db-07a0-472d-a8f0-fb41081d8d5d





## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> N/A
